src/plugins/intel_cpu/CMakeLists.txt

```diff
@@ -287,14 +302,14 @@ cross_compiled_file(${TARGET_NAME}
         NAMESPACE   ov::Extensions::Cpu::XARCH
 )
 cross_compiled_file(${TARGET_NAME}
-        ARCH AVX512F AVX2 NEON_FP16 ANY
+        ARCH ${SOFTMAX_ARCH_LIST}
```

Comment:
Some tests are still failing: https://github.com/openvinotoolkit/openvino/actions/runs/12275233004/job/34252933475?pr=27273.
Now it is caused by the fact you changed the order of NEON_FP16 and ANY arch in that list. Please restore original order .

Reply:
Fixed in latest commit.